docs/skopeo.1.md

```diff
@@ -47,7 +47,7 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   **oci-archive:**_path_**:**_tag_
```

Comment:
@TomSweeneyRedHat does this need to be merged before a Skopeo release? At least this part shows up in the shipped man page.

Reply:
Yes, I think it should be and @rhatdan has made it so.